third-culture-software/bhima
# Small fixes for asset entry

This PR addresses a couple small issues with entering assets (via the Stock Entry page).
- When an inventory item is an asset, it forces the global quantity and the lot quantity to be 1 (since assets can only be individual/single lots).   It also makes the global quantity uneditable so the user cannot attempt to add more than quantity=1 on the lot entry line.
- Fixes an issue with a misplaced file.

This PR addresses the issue described in https://github.com/IMA-WorldHealth/bhima/pull/6417#pullrequestreview-891966667.
Closes https://github.com/IMA-WorldHealth/bhima/issues/6420

**TESTING**
- Follow the steps in https://github.com/IMA-WorldHealth/bhima/pull/6417#issue-1147884432 to create an asset inventory group and item.
- Go to:  Stock > Stock Entry
  - Try entering stock and adding an asset for the inventory class defined above.  Note that the global quantity is set to 1 and cannot be changed.
  - Add a second instance of the same type of asset
  - Complete the asset entry.
- Go to the Stock > Lots registry.  Update the displayed columns to show asset related fields (is_asset, manufacturer brand/model, serial number) and verify the newly entered assets are present.
